perun-db/postgres.sql

```diff
@@ -563,9 +563,9 @@ create table application_data (
 create table application_mails (
 	id integer not null,
 	form_id integer not null,       --identifier of form (application_form.id)
-	app_type varchar(30) not null,  --application type (initial/extension)
-	mail_type varchar(30) not null, --type of mail (user/administrator)
-	send varchar(1) not null,       --sent (Y/N)
+	app_type varchar not null,  --application type (initial/extension)
+	mail_type varchar not null, --type of mail (user/administrator)
+	send char(1) not null,       --sent (Y/N)
```

Comment:
Again this change is missing from postgresql changelog.

Reply:
The same, no need to migrate it in existing databases.